content/desktop/synchronized-file-sharing.md

```diff
@@ -1,73 +1,55 @@
 ---
-title: Synchronized file sharing
-description: Get started with Synchronized file sharing on Docker Desktop.
-sitemap: false
+title: Synchronized File Shares
+description: Get started with Synchronized File Shares on Docker Desktop.
+keyword: mutagen, file sharing, docker desktop, bind mounts
 ---
 
-> **Early Access**
+> **Note**
 >
-> Synchronized file sharing is an early-access feature. 
->
-> If you would like to be considered for Synchronized file sharing testing, you can
-> [sign up for the early access program](https://www.docker.com/docker-desktop-preview-program/).
-{ .restricted }
+> Synchronized File Shares is available for customers with a Docker subscription. 
 
-Synchronized file sharing is an alternative file sharing mechanism powered by [Mutagen](https://mutagen.io/). It provides fast and flexible host-to-VM file sharing by replacing bind mounts with synchronized filesystem caches. 
+Synchronized File Shares is an alternative file sharing mechanism powered by [Mutagen](https://mutagen.io/). It provides fast and flexible host-to-VM file sharing by replacing bind mounts with synchronized filesystem caches. 
 
 ![Image of Synchronized file shares pane](images/synched-file-sharing.png)
  
 ## Who is it for?
 
-Synchronized file sharing is for developers who: 
+Synchronized File Shares is for developers who: 
```

Comment:
Maybe we could say "is ideal for developers who:"?  I don't necessarily want to make people think this won't be helpful to them if they don't have the world's largest monorepo; there could be pathological cases that benefit from this functionality with smaller projects.